Update for Oauth destinations (#1028)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/campaign-manager-360.mdx

## Connection settings

To successfully configure Campaign Manager 360 as a destination, first authenticate your account by following the below steps:

1. Click **Create Account** in the **Account Settings** section.

> 2. Select **Sign in with Google** from the modal.

Reviewer: ```suggestion
2. In the resulting modal, click the **Sign in with Google** button.
```

3. Choose the required account and grant RudderStack the required permissions.
4. Click **Save** to use the specified account, as shown:

<img src="../../assets/event-stream-destinations/account-connect-normal.png" alt="Google Account authentication" />

<div class="infoBlock">

> In case you have authenticated multiple accounts, you can click <strong>Edit Credentials</strong> to select/delete any other authenticated account, as shown:

Reviewer: ```suggestion
In case you have authenticated multiple accounts, you can click <strong>Edit Credentials</strong> to select or delete any other authenticated account, as shown:
```

<br/><br/>

<img src="../../assets/event-stream-destinations/edit-account-creds.png" alt="Google Account authentication" />

> RudderStack will <strong>throw an error</strong> if you try to delete an account which is being used by any other connection set up for the same destination.

Reviewer: ```suggestion
RudderStack will <strong>throw an error</strong> if you try to delete an account used by any other connection set up for the same destination.
```
